### PR TITLE
Add exemplar tags

### PR DESCRIPTION
**Pull request by neuromechanist**
I have added a couple of tags with references in the Annotation column.
I think there are some outstanding points:
1. Should the database names also defined in the prefixes (I assume not, because the names are already available in OBO)
2. Should we use `has_dbxref` or `db_xref` for cross-referencing (or any other label)? `has_dbxref` is defined [here](http://www.geneontology.org/formats/oboInOwl#dbxref) and `db_xref` is defined [here](https://www.ncbi.nlm.nih.gov/genbank/collab/db_xref/).
3. What should I do for the abbreviations, like `LHJ` and `RHJ`? Many of them are quite well-known and handy to use

### CODE REVIEW

File: library_schemas/slam/prerelease/hedtsv/HED_slam_1.0.0/HED_slam_1.0.0_Tag.tsv [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+	Anatomical-landmark	0	Anatomical-item	rooted=Anatomical-item	A collection of specific points on the human body used for anatomical and sensor placement reference.		dc:source "Original", rdfs:comment "Body landmarks are used to standardize sensor placement.
+	Head-landmark	1	Anatomical-landmarks		Points on the head used as reference landmarks.		dc:source "Original"
+	Nasion	2	Head-landmark		The midpoint between the eyes just above the bridge of the nose.		
+	Inion	2	Head-landmark		The most prominent point of the occipital bone at the back of the head.		
+	Left Helix-Tragus Junction (LHJ)	2	Head-landmark		The point where the helix meets the tragus of the left ear.		
+	Right Helix-Tragus Junction (RHJ)	2	Head-landmark		The point where the helix meets the tragus of the right ear.		
+	Vertex	2	Head-landmark		The highest point on the head in the midline.		
+	Mastoid-process	2	Head-landmark				dc:source "Original"
+	Torso-chest-landmark	1	Anatomical-landmarks		Points on the chest used as reference landmarks.		dc:source "Original"
+	Acromion-process	2	Torso-chest-landmark		The outermost point of the shoulder blade.		has_dbxref: "NCIT:C32048" "FMA:12525" "UBERON:0002497"
```
Comment (VisLab) [inline library_schemas/slam/prerelease/hedtsv/HED_slam_1.0.0/HED_slam_1.0.0_Tag.tsv:11]: The syntax here is incorrect.  Assuming that we were using obo: as the namespace prefix and `has_dbxref` as the AnnotationProperty, the syntax would be:

obo:has_dbxref  "NCIT:C32048",  obo:has_dbxref "FMA:12525", obo:has_dbxref  "UBERON:0002497"

However, we need to do some research on exactly what AnnotationProperty we want to use --- this is one that everyone will want to use.

**Review by VisLab — CHANGES_REQUESTED** [review]
This is a good first start. I am pretty sure that the links we will want to add will all be AnnotationProperty types (like has_dbxref) so we won't need any different infrastructure.

I have commented on the syntax problems above.  After we finalize exactly which thing we will use, this will just need the syntax corrected.

**Comment by neuromechanist** [comment]
Thanks Kay,

I corrected the Annotations and added obo to the prefix table.

File: library_schemas/slam/prerelease/hedtsv/HED_slam_1.0.0/HED_slam_1.0.0_Prefixes.tsv [hunk 2]
```diff
@@ -8,4 +8,5 @@ prov:	http://www.w3.org/ns/prov#	The PROV namespace [PROV-DM]
 dc:	http://purl.org/dc/elements/1.1/	The Dublin Core elements
 terms:	http://purl.org/dc/terms/	The Dublin Core terms
 foaf:	http://xmlns.com/foaf/0.1/	Friend-of-a-Friend http://xmlns.com/foaf/spec/
-has_dbxref	http://www.geneontology.org/formats/oboInOwl#dbxref	Relates an entity to an external database entry
\ No newline at end of file
+obo:	http://purl.obolibrary.org/obo/	Open Biomedical Ontologies
+has_dbxref:	http://www.geneontology.org/formats/oboInOwl#dbxref	Relates an entity to an external database entry
```
Comment (VisLab) [inline library_schemas/slam/prerelease/hedtsv/HED_slam_1.0.0/HED_slam_1.0.0_Prefixes.tsv:12]: The has_debxref should not be in the Prefixes file  it should be in the AnnotationsExternal file -- it is not a namespace, it is an AnnotationProperty.  Thus Line 12 should not be there -- However the obo ref should maybe be http://www.geneontology.org/formats/oboInOwl#.  Again, we will need to think about what property we want to use in general for this -- the `has_dbxref`  has a specific meaning.

**Review by VisLab — CHANGES_REQUESTED** [review]
There are some circular or incorrect references and other syntax errors:
- `Head-landmark` lists its parent as `Anatomical-landmarks`. It should be `Anatomical-landmark`
- `Torso-chest-landmark`  also lists `Anatomical-landmarks` as it parent.
- HED tag names cannot have blanks or parentheses. By convention only first letter is capitalized.
   - `Left-helix-tragus-junction`
   - `Right-helix-tragus-junction`
   - `Vertebra-promines`
   
 The stuff in parentheses should probably go in the description.

**Comment by neuromechanist** [comment]
> There are some circular or incorrect references and other syntax errors:
> 
> * `Head-landmark` lists its parent as `Anatomical-landmarks`. It should be `Anatomical-landmark`
> * `Torso-chest-landmark`  also lists `Anatomical-landmarks` as it parent.
> * HED tag names cannot have blanks or parentheses. By convention only first letter is capitalized.
>   
>   * `Left-helix-tragus-junction`
>   * `Right-helix-tragus-junction`
>   * `Vertebra-promines`
> 
> The stuff in parentheses should probably go in the description.

Addressed!

**Review by VisLab — APPROVED** [review]

**Pull request merged**